### DISCUSSION TITLE
Overflow error

### DISCUSSION THREAD

**Discussion by flubber0429** [globaleaks/globaleaks-whistleblowing-software#3909] [Q&A] [answered]
When the manager logs in and tries to access the list of reports, the screen appears blank and I receive error emails of this type:
OverflowError Result too large to be represented.
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/twisted/python/threadpool.py", line 244, in inContext
    result = inContext.theWork()  # type: ignore[attr-defined]
            ^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/twisted/python/threadpool.py", line 260, in <lambda>
    inContext.theWork = lambda: context.call(  # type: ignore[attr-defined]
                                
globalaks is installed on an exclusive debian 12 server in an internal nat network

**Comment by evilaliv3** [comment]
On which type of architecture is this system based?

**Comment by flubber0429** [comment]
Virtual machine on  vmware. Linux is debian 12.
Server is on NAT, but thera are not firewall rules on this server

**Comment by evilaliv3** [comment, marked as answer]
Is it possible that this setup is not a 64bit setup?

**Comment by flubber0429** [comment]
You are right. I checked and they installed a 32 bit debian

**Reply by evilaliv3** [reply]
Ok, install a debian 64bit and all should work like a charm!
